darwin/torch/transforms.py

```diff
@@ -349,7 +363,7 @@ def _pre_process(self, image: np.ndarray, annotation: dict) -> dict:
             albumentation_dict["labels"] = labels.tolist()
 
         masks = annotation.get("masks")
-        if masks is not None:
+        if masks is not None and masks.numel() > 0:  # using numel() to check if tensor is non-empty
```

Comment:
if masks is None this will error

Reply:
Python evaluates from left to right, since it is a AND and not an OR it will check if it is None and then return false (since it does not need to check the statement to the right).

Reply:
oh nice good to know!